tests/integration/common/checks-after-conversion/main.fmf

```diff
@@ -157,3 +157,8 @@ order: 52
     description+: |
         Verify that firewalld is not reporting any issues in the logs.
     test: pytest -m test_check_firewalld_errors
+
+/check_repoquery_empty_exclude:
+    summary+: |
+        Verify that convert2rhel used `--setopt=exclude= ` in every `repoquery` call.
+    test: pytest -m test_check_repoquery_empty_exclude
```

Comment:
Since we're just reading from the log, can't we just fit it [under here ](https://github.com/oamg/convert2rhel/pull/1346/files#diff-cc849537e44530b2b2d10af1f12d93997ca4371954b354a4104f1639d186da5aR106)?

Reply:
I guess that is a good idea!